C/Bit--Manipulation/Swap_even_and_odd_bits.c

```diff
@@ -0,0 +1,36 @@
+#include <stdio.h>
```

Comment:
Add a description of the program.

Reply:
Yes, I have done it. Now, you can review my code again.

Reply:
No. You have not resolved it yet

Reply:
> No. You have not resolved it yet

But, I have resolved it and in the commits section, it is showing. Could you tell me any reason why it is not showing the changes?